app/api/sessions.py

```diff
@@ -104,12 +106,24 @@ def validate_date(self, data, original_data):
         type_='speaker')
 
 
-class SessionList(ResourceList):
+class SessionListPost(ResourceList):
     """
-    List and create Sessions
+    List Sessions
     """
     def before_post(self, args, kwargs, data):
         require_relationship(['event'], data)
+        if not has_access('is_coorganizer', event_id=data['event']):
```

Comment:
> Session can have a state of either draft or pending
Cannot add/modify track/microlocation relation

Should a 403 be raised in this case or a 422, or should it be ignored i.e relationships will be ignored and state will be changed to draft